cosmo_tester/test_suites/manager_of_managers/mom_test.py

```diff
@@ -13,62 +13,105 @@
 from cosmo_tester.test_suites.snapshots import restore_snapshot
 
 from . import constants
-from .tier_1_clusters import FloatingIpTier1Cluster, FixedIpTier1Cluster
+from .regional_cluster import (
+    FloatingIpRegionalCluster,
+    FixedIpRegionalCluster)
 
 
 fixed_ip_clusters = []
 floating_ip_clusters = []
 
-tier_2_hosts = None
+central_hosts = None
+ATTRIBUTES = util.get_attributes()
 
 
-# Using module scope here, in order to only bootstrap one Tier 2 manager
+# Using module scope here, in order to only bootstrap one Central manager
 @pytest.fixture(scope='module')
-def tier_2_manager(cfy, ssh_key, module_tmpdir, attributes, logger):
+def central_manager(cfy, ssh_key, module_tmpdir, attributes, logger):
     """
-    Creates a Tier 2 Cloudify manager with all the necessary resources on it
+    Creates a Central Cloudify manager with all the necessary resources on it
     """
-    global tier_2_hosts
-    tier_2_hosts = TestHosts(
+    global central_hosts
+    central_hosts = TestHosts(
             cfy, ssh_key, module_tmpdir, attributes, logger)
-    tier_2_hosts.create()
-    manager = tier_2_hosts.instances[0]
+    central_hosts.create()
+    manager = central_hosts.instances[0]
     manager.use()
-    _upload_resources_to_tier_2_manager(cfy, manager, logger)
+    _upload_resources_to_central_manager(cfy, manager, logger)
     yield manager
 
     # We don't need to teardown - this is handled by `teardown_module`
 
 
-def _upload_resources_to_tier_2_manager(cfy, manager, logger):
+def _upload_resources_to_central_manager(cfy, manager, logger):
+    root_resource_dir = os.path.dirname(__file__)
+
+    license_path = os.path.abspath(
+        os.path.join(
+            root_resource_dir,
+            '..', '..', 'resources/test_valid_paying_license.yaml'))
+    manager.put_remote_file('/etc/cloudify/license.yaml', license_path)
+    manager.run_command(
+        'sudo chown cfyuser:cfyuser '
+        '/etc/cloudify/license.yaml')
+
+    wagon_save_path = os.path.abspath(
+        os.path.join(
+            root_resource_dir, constants.MOM_PLUGIN_WGN_NAME))
+
+    util.download_asset(
+        constants.MOM_PLUGIN_REPO_PATH,
+        constants.MOM_PLUGIN_RELEASE_NAME,
+        constants.MOM_PLUGIN_WGN_NAME,
+        wagon_save_path,
+        os.environ.get('SPIRE_GIT_TOKEN')
+    )
+
+    plugin_yaml_save_path = os.path.abspath(
+        os.path.join(
+            root_resource_dir, 'plugin.yaml'))
+
+    util.download_asset(
+        constants.MOM_PLUGIN_REPO_PATH,
+        constants.MOM_PLUGIN_RELEASE_NAME,
+        'plugin.yaml',
+        plugin_yaml_save_path,
+        os.environ.get('SPIRE_GIT_TOKEN')
+    )
+
     cfy.plugins.upload(
-        constants.MOM_PLUGIN_WGN_URL,
-        '-y', constants.MOM_PLUGIN_YAML_URL
+        wagon_save_path,
+        '-y', plugin_yaml_save_path
     )
     cfy.plugins.upload(
         constants.OS_PLUGIN_WGN_URL,
         '-y', constants.OS_PLUGIN_YAML_URL
     )
 
+    manager_install_rpm = \
```

Comment:
Maybe move this to util func

Reply:
The util function doesn't check the ATTRIBUTES